geode-docs/tools_modules/gfsh/command-pages/start.html.md.erb

```diff
@@ -526,7 +538,11 @@ start server --name=value [--assign-buckets(=value)] [--bind-address=value]
 </tr>
 <tr>
 <td><span class="keyword parmname">\-\-bind-address</span></td>
-<td>The IP address on which the server will be bound.</td>
+<td>The IP address on which the server will be bound. The '*' character is accepted for wildcard addresses.
+<div class="note note">
+<b>Note:</b> In case the server is bound to all local addresses, UDP membership-related traffic will be bound to the local host machine's address.
```

Comment:
Slight rewrite to use present tense and clarify default:
When the server is bound to all local addresses, the default for UDP membership-related traffic is to bind to the local host machine's address.